lib/bundle/commands/check.rb

```diff
@@ -14,54 +16,75 @@ def reset!
         Bundle::BrewServices.reset!
       end
 
+      def exit_on_first_error?
+        Bundle::Checker.exit_on_first_error?
+      end
+
+      def output_errors?
+        ARGV.include?("--verbose")
+      end
+
       def run
-        if any_taps_to_tap? ||
-           any_casks_to_install? ||
-           any_apps_to_install? ||
-           any_formulae_to_install? ||
-           any_formulae_to_start?
+        @dsl ||= Bundle::Dsl.new(Bundle.brewfile)
+
+        checks = {
+          taps_to_tap: "Taps",
+          casks_to_install: "Casks",
+          apps_to_install: "Apps",
+          formulae_to_install: "Formulae",
+        }
+        check_method_names = checks.keys
+
+        completed_checks = []
+        errors = []
+        enumerator = exit_on_first_error? ? :any? : :map
+
+        work_to_be_done = check_method_names.send(enumerator) do |check_method|
+          check_errors = send(check_method)
+          completed_checks << check_method
+          any_errors = check_errors.any?
+          errors.concat(check_errors) if any_errors
+          any_errors
+        end
+
+        work_to_be_done = work_to_be_done.any? if work_to_be_done.class == Array
+
+        if work_to_be_done
           puts "brew bundle can't satisfy your Brewfile's dependencies."
+          unchecked_checks = (checks.keys - completed_checks)
+          completed_checks.each { |checked| puts "#{checks[checked]} were checked." }
```

Comment:
Let's not output these in non-verbose mode.

Reply:
✅